tailwind.config.js

```
@@ -82,7 +82,9 @@ export default {
         's-28': '28px',
         's-17': '17px',
         's-15': '15px',
+        's-12': '12px',
```

Biome (1.9.4)

[error] This property value named s-12 is later overwritten by an object member with the same name.

Overwritten with this value.

If an object property with the same name is defined multiple times (except when combining a getter with a setter), only the last definition makes it into the object and previous definitions are ignored.
Unsafe fix: Remove this property value named s-12


(lint/suspicious/noDuplicateObjectKeys)